atf_eregs/static/regulations/css/less/typography.less

```diff
@@ -392,6 +392,10 @@ The `<ol>` tag needs to have the appropriate list type applied
 ```
 */
 
+.main-content {
+    overflow-x:scroll;
+}
```

Comment:
I think so - I will make that change so it is in the next push.  In upcoming versions I think it should be namespaced a little better to ensure it only applies to ereg-based tables / content.